client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-category-list.tsx

```diff
@@ -114,7 +114,11 @@ const ScreenCategoryList = ( {
 							} }
 						>
 							<span>{ label }</span>
-							<Icon icon={ chevronRight } size={ 24 } />
+							<Icon
```

Comment:
Can we just use `<Gridicon icon="chevron-right" size={ 24 } />`? Are there any reasons that it's using `Icon`? 
@Automattic/lego

Reply:
Calypso uses both `@wordpress/icon` and `Gridicon` and I'm not sure whether we have any preference or not. I'd prefer to use `@wordpress/icon` if we can find the icon we want (or maybe I'm wrong).

But I think there is a good reason to use `@wordpress/icon` here since we want to align the UI of the Pattern Assembler with the Site Editor, so I guess it might be better to keep using `@wordpress/icon` even though they might look the same. Does it make sense?

Reply:
> But I think there is a good reason to use @wordpress/icon here since we want to align the UI of the Pattern Assembler with the Site Editor, so I guess it might be better to keep using @wordpress/icon even though they might look the same.

That makes sense to me! Thank you for letting me know!